src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -462,8 +462,10 @@ Properties needed for the deployment of instance are:
    -   *image:* Defines which image will be used to create the OpenStack instance. The value to provide is the unique image Id.
    
    -   *flavor:* Defines the size of the instance. The value to provide is the flavor name or the flavor Id. For more information see https://docs.openstack.org/openstack-ops/content/flavors.html[OpenStack flavors].
+
+   -   *networkId:*  Defines the identifier of the network to be attached to the instance. This value can be blanked to leave the selection to OpenStack.
```

Comment:
can be blanked => can be blank

Reply:
multiple network Ids could be provided? Why not mention that in the doc

Reply:
@mboussaa : Noop, that's not the case here.

Reply:
ok